examples/android/libKtAndroid/BUILD.bazel

```diff
@@ -19,12 +19,23 @@ java_plugin(
     deps = ["@maven//:com_google_auto_value_auto_value"],
 )
 
+kt_ksp_plugin_info(
```

Comment:
Why not `kt_ksp_plugin`?

Reply:
Ah good catch. Renamed this to just `kt_ksp_plugin`